src/components/templates/PopoverTemplate/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {
+  Button,
+  Placement,
+  Popover,
+  PopoverBody,
+  PopoverCloseButton,
+  PopoverContent,
+  PopoverFooter,
+  PopoverHeader,
+  PopoverTrigger,
+} from '@chakra-ui/react';
+
+type PopoverProps = {
+  header: string;
+  triggerText: string;
+  variant?: string;
+  showFooter?: boolean;
+  placement?: Placement;
+  readonly body: React.ReactNode;
+  readonly footer: React.ReactNode;
+};
+
+const PopoverTemplate = ({ header, triggerText, variant, showFooter, placement, body, footer }: PopoverProps) => (
```

Comment:
Very nice template :) 
I'm a little concerned about how it will look if a consumer passes a large component inside, but currently let's not worry about that and use it "sensibly" :)